crossbundle/cli/src/commands/mod.rs

```diff
@@ -5,6 +5,7 @@ pub mod new;
 pub mod run;
 
 use crate::error::Result;
+pub use build::*;
```

Comment:
Bad practice. Better replace with `pub mod`.

Reply:
We have `pub mod build` yet. If delete `pub use build::*;` I will not import needed functions `pub use commands::{android, BuildContext, Commands, SharedBuildCommand};`